pkg/util/system/common_linux.go

```diff
@@ -31,11 +31,56 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+	"sync"
+	"syscall"
+	"time"
 	"unicode"
 
+	"github.com/cakturk/go-netstat/netstat"
 	"k8s.io/klog/v2"
 )
 
+var (
+	portToPidMap = sync.Map{}
```

Comment:
better to use atomic value, since data maybe left over cause error if kubelet change the port